plugins/generator-1.19.4/forge-1.19.4/utils/triggers.java.ftl

```diff
@@ -217,6 +217,19 @@
 </#if>
 </#macro>
 
+<#macro piglinNeutral procedure="">
+    <#if procedure?has_content || hasProcedure(procedure)>
+	@Override public boolean makesPiglinsNeutral(ItemStack stack, LivingEntity entity) {
+        <#if hasProcedure(procedure)>
+			double x = entity != null ? entity.getX() : 0.0;

```

Comment:
Can this entity actually be null as it is not annotated as nullable in the code of Minecraft per my checks?

Reply:
You're right, it cannot be null, I'm fixing this